### PR TITLE
Update jandex gradle plugin version in docs

### PR DESCRIPTION
**Pull request by glefloch**
This update the version of jandex in documentation following the update.

### CODE REVIEW

File: docs/src/main/asciidoc/cdi-reference.adoc [hunk 1]
```diff
@@ -66,16 +69,34 @@ To generate the index just add the following to your `pom.xml`:
...
+[role="secondary asciidoc-tabs-sync-gradle-groovy"]
+.Gradle (Groovy DSL)
+****
 
-[source,groovy]
+[source,groovy,subs=attributes+]
 ----
 plugins {
-    id 'org.kordamp.gradle.jandex' version '0.11.0'
+    id 'org.kordamp.gradle.jandex' version '0.13.0'
```
Comment (gsmet) [inline docs/src/main/asciidoc/cdi-reference.adoc:81]: Could we make it an attribute here: https://github.com/quarkusio/quarkus/blob/main/docs/pom.xml#L2857 ?

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks! I added a small suggestion.

File: docs/src/main/asciidoc/cdi-reference.adoc (hunk 1 above)
Comment (glefloch) [reply docs/src/main/asciidoc/cdi-reference.adoc:81]: Right, definitely!

**Comment by glefloch** [comment]
@gsmet I updated the branch

**Comment by glefloch** [comment]
@gsmet CI seems to timeout, any idea?

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
@glefloch no worries, it happens from time to time.

**Pull request merged**